RELEASE/scripts/autoscend/auto_buff.ash

```diff
@@ -731,12 +740,15 @@ boolean buffMaintain(effect buff, int mp_min, int casts, int turns, boolean spec
 			useSkill = $skill[Tenacity of the Snapper];
 		}																						break;
 	case $effect[There is a Spoon]:				useItem = $item[Dented Spoon];					break;
+	case $effect[They\'ve Got Fleas]:			useItem = $item[Out-of-work circus flea];		break;
 	case $effect[This is Where You\'re a Viking]:useItem = $item[VYKEA woadpaint];				break;
 	case $effect[Throwing Some Shade]:			useItem = $item[Shady Shades];					break;
 	case $effect[Ticking Clock]:				useItem = $item[Cheap wind-up Clock];			break;
+	case $effect[Tingling Insides]:				useItem = $item[electric mushroom];				break;
```

Comment:
This should get added to auto_restore. To find the spot, search for
`buffMaintain($effect[Tingly Tongue]);`

Reply:
Honestly it is getting to have quite a few entries now. Feels like we will use them all at once, have super great MP regen for a few turns, then run out. Would be better to spread them out somehow. Could be a `for each` type structure and breaks out when any item is actually used. So each time we have to restore it only pops one potion.

Totally fine to simply add the tingling insides effect for now and leave the algorithm change to a future PR